### PR TITLE
Stations view

### PR DESCRIPTION
**Pull request by cgalvarino**
@lukecampbell see if this gets us off the ground floor

### CODE REVIEW

**Comment by cgalvarino** [comment]
Not super pretty, but the metadata panel is tied to a collection and is rendered on a station select (both from TOC and map click).
stations

File: habs_portal/static/js/partials/StationMetadata.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<% collection.each(function(model) { %>
```
Comment (cgalvarino) [inline habs_portal/static/js/partials/StationMetadata.html]: @lukecampbell I'm not sure if this proper backbone etiquette, but I'm sending a collection to the metadata panel even though this collection will only have most 1 record (the active station).

Comment (lukecampbell) [inline habs_portal/static/js/partials/StationMetadata.html]: this is how I do it as well. Not sure if it's backbone-ish but it works and it's very understandable.

**Comment by lukecampbell** [comment]
Nothing a little CSS can't fix :+1:

**Pull request merged**